workloads/templates/common-templates.md

```diff
@@ -131,3 +131,46 @@ oc process rhel7-generic-tiny PVCNAME=mydisk NAME=rheltinyvm | oc apply -f -
 
 Please note that, after the generation step, VM objects and template objects have no relationship with each other besides the aforementioned label (e.g. changes
 in templates do not automatically affect VMs, or vice versa).
+
+## common template customization
+
+### template customization - memory, CPU
+
+There are three options to customize VM memory and CPU:
```

Comment:
Your title says template customization, not VM customization. Selecting a flavor is not really relevant as that usually means a different template file.

Reply:
I think we can remove the flavor references, and in general the content related to the VM customization.